src/LibHdb++Cassandra.cpp

```diff
@@ -875,19 +1025,24 @@ void HdbPPCassandra::extract_and_bind_state(CassStatement* statement,
 	}
 	else
 	{
-		cerr << __func__ << "extract_read failed for attribute "<< data->attr_name << "! (" << __FILE__ << ":" << __LINE__ << ")" << endl;
+		// TODO (SJ) Should we be doing something with these?
+		ERROR_MSG("extract_read failed for attribute "<< data->attr_name << endl);
```

Comment:
I think if we are in this part of the code, it means the attribute value received is not from the type we are expecting and the new type is not compatible with the old one.
This could happen if an attribute type was changed since it was added into att_conf table from HDB++ Cassandra table.
So there is a mismatch between the real attribute type and the data_type configuration in att_conf table.
In this case, the HDB++ admin should be notified because some manual intervention will be required so I think the error should be reported to the user... Unless I am misinterpreting the code or forgetting another use case where this error could happen.